app/api/suggestions/suggestions.ts

```diff
@@ -339,4 +346,179 @@ export const Suggestions = {
       }
     }
   },
+
+  deleteSuggestionsNotConfigured: async (
```

Comment:
It looks like many of this new functions are not meant to be exported. They shouldn't be declared in the Suggestions object.

In fact, it looks like only saveConfigurations should be in the object to be exported.

Reply:
Done.